frappe/integrations/doctype/webhook/__init__.py

```diff
@@ -37,7 +37,7 @@ def run_webhooks(doc, method):
 
 	def _webhook_request(webhook):
 		if not webhook.name in frappe.flags.webhooks_executed.get(doc.name, []):
-			frappe.enqueue("frappe.integrations.doctype.webhook.webhook.enqueue_webhook", doc=doc, webhook=webhook)
+			frappe.enqueue("frappe.integrations.doctype.webhook.webhook.enqueue_webhook", enqueue_after_commit=False, doc=doc, webhook=webhook)
```

Comment:
enqueue_after_commit should be True!!!

Reply:
Oops! 
:slightly_frowning_face: 

Thanks.